cloudmetrics/metrics/woi.py

```diff
@@ -5,6 +5,22 @@
 import matplotlib.pyplot as plt
 import pywt
 
+_CACHED_VALUES = dict()
+
+
+def _get_swt(cloud_scalar, pad_method, wavelet, separation_scale):
```

Comment:
:) We can put this kind of thing into a decorator at some point. I would use `functools.lru_cache` but that doesn't work with just numpy arrays because they can't always be serialized.